pkg/oauth/registry/oauthaccesstoken/etcd/etcd.go

```diff
@@ -0,0 +1,55 @@
+package etcd
+
+import (
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kerrs "github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/fields"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/registry/generic"
+	etcdgeneric "github.com/GoogleCloudPlatform/kubernetes/pkg/registry/generic/etcd"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/tools"
+
+	"github.com/openshift/origin/pkg/oauth/api"
+	"github.com/openshift/origin/pkg/oauth/registry/oauthaccesstoken"
+)
+
+// rest implements a RESTStorage for access tokens against etcd
+type REST struct {
+	etcdgeneric.Etcd
+}
+
+const EtcdPrefix = "/registry/oauth/accesstokens"
```

Comment:
While we're tidying up, how about creating a global constant for a prefix on your prefix.  In this case, it would be "/registry", but it seems like we ought to move our stuff under "/openshift" or some such.